embulk-input-mysql/src/main/java/org/embulk/input/mysql/getter/AbstractMySQLTimestampIncrementalHandler.java

```diff
@@ -9,22 +9,22 @@
 import org.embulk.spi.Exec;
 import org.embulk.spi.time.TimestampFormatter;
 import org.embulk.spi.time.TimestampParser;
-import org.joda.time.DateTimeZone;
 
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Timestamp;
+import java.time.ZoneId;
 import java.util.Optional;
 
 public abstract class AbstractMySQLTimestampIncrementalHandler
         extends AbstractIncrementalHandler
 {
-    protected final DateTimeZone sessionTimeZone;
+    protected final ZoneId sessionTimeZone;
```

Comment:
I have a concern here, a `ZoneId` could have many `ZoneOffset`, should we use ZoneOffset in this?